presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/UnaliasSymbolReferences.java

```diff
@@ -694,6 +694,14 @@ private VariableReferenceExpression canonicalize(VariableReferenceExpression var
             return new VariableReferenceExpression(canonical, variable.getType());
         }
 
+        private Optional<VariableReferenceExpression> canonicalize(Optional<VariableReferenceExpression> variable)
+        {
+            if (variable.isPresent()) {
```

Comment:
```java
return variable.map(this::canonicalize);

Reply:
Same as above. I used the same code logic as the symbol version. It's probably not the best style but I'd rather keep them looking the same in the refactoring.